From b64e4fca117f8e6cab7d816e728da8e39b17b960 Mon Sep 17 00:00:00 2001
From: "jian.xu" <jian.xu@amlogic.com>
Date: Wed, 20 Aug 2014 12:05:24 +0800
Subject: [PATCH 4793/5965] PD#89468 : fix resume/play video freeze issue,as
 when resume play,apts discontinue happened,in this case a/v pts diff >
 120s,it cause apts no chance to set scr

Change-Id: I1130eb9bb7b4694317187afe9fad7123d68ac444
---
 drivers/amlogic/amports/tsync.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/amports/tsync.c b/drivers/amlogic/amports/tsync.c
index 0c43d022abc2..6a6db54c8366 100755
--- a/drivers/amlogic/amports/tsync.c
+++ b/drivers/amlogic/amports/tsync.c
@@ -509,7 +509,7 @@ void tsync_avevent_locked(avevent_t event, u32 param)
         /*set tsync mode to vmaster to avoid video block caused by avpts-diff too much
           threshold 120s is an arbitrary value*/  
         t = abs(timestamp_apts_get()-timestamp_vpts_get())/TIME_UNIT90K;
-        if (tsync_enable && !get_vsync_pts_inc_mode() && t<120) {
+        if (tsync_enable && !get_vsync_pts_inc_mode()/* && t<120*/) {
             tsync_mode = TSYNC_MODE_AMASTER;
         } else {
             tsync_mode = TSYNC_MODE_VMASTER;
-- 
2.19.0

